src/Grpc.Net.ClientFactory/Internal/GrpcCallInvokerFactory.cs

```diff
@@ -40,15 +40,15 @@ public GrpcCallInvokerFactory(ILoggerFactory loggerFactory)
             _loggerFactory = loggerFactory;
         }
 
-        public CallInvoker CreateCallInvoker(HttpClient httpClient, string name, GrpcClientFactoryOptions clientFactoryOptions)
+        public CallInvoker CreateCallInvoker(HttpMessageHandler httpHandler, string name, Type type, GrpcClientFactoryOptions clientFactoryOptions)
```

Comment:
This is a breaking change isn't it?

Reply:
What do you mean? GrpcCallInvokerFactory is internal so its not a public API breaking change

Reply:
Duh... I for some reason thought this was public API.